siteConfig.js

```diff
@@ -121,6 +121,13 @@ const schema = {
     docExample: 'transforms:\n  email: "md5" # The email field will be MD5-hashed',
     format: Object,
     default: {}
+  },
+  reCaptcha: {
+    enabled: {
+      doc: 'Set to `true` to force reCAPTCHA validation, set to `false` to accept comments without reCAPTCHA.',
+      format: Boolean,
+      default: true
```

Comment:
Already fixed it, but forget to push the commit. I'll push it with the other fixes.